### ISSUE TITLE
same set of videos are returned on each reload

### ISSUE THREAD

**Issue by Khateebxtreme** [adrianhajdin/project_youtube_clone#22]
I came across an issue where on my homepage / search results page on each reload or query returns same set of videos and no new videos are rotated in that pool

**Comment by raccoonwannafly** [comment]
also adrian used an outdated api, try tweaking the app to use the earliest version, its called alternate or alternative v3 I think

**Comment by Khateebxtreme** [comment]
Yeah, I fixed the code in my local to display random search results based on various parameters.

**Comment by Akhil-Sharma-26** [comment]
> Yeah, I fixed the code in my local to display random search results based on various parameters.

Can You please share how you did that?

**Comment by BijendraRaut** [comment]
Hey Akhil, I think I can help you with that.

**Comment by Alea-Tennu** [comment]
> Yeah, I fixed the code in my local to display random search results based on various parameters.

Please do you mind sharing your fix?

**Comment by Khateebxtreme** [comment]
Sorry for not responding sooner as I didn't get any notification for this thread.

My fix was not reliant on the API used in this project but I had to shift my concern over to Youtube Data API which is the parent API of Youtube V3 and I had to randomize the values of some parameters (can be found when reading the API docs) when pulling up the videos. The number of parameters and instructions are limited in the documentation section of Youtube V3 API which limits the current project to generate the same set of videos.
